app/news/[id]/page.tsx

```
@@ -1,195 +1,201 @@
+                <div className="mb-6">
+                  <p className="text-lg font-medium text-gray-800">
+                    {currentNews.summary}
                   </p>
                 </div>
               )}
-              <div className='prose max-w-none'>
+              <div className="prose max-w-none dom-purify-content">
                 <div
                   dangerouslySetInnerHTML={{
-                    __html: DOMPurify.sanitize(latestNews.content ?? ''),
+                    __html: DOMPurify.sanitize(currentNews.content ?? ""),
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)